### ISSUE TITLE
Image dropout enhancement?

### ISSUE THREAD

**Issue by AI-Casanova** [kohya-ss/sd-scripts#175]
I've been exploring a way of training a person with limited data, in which I use more/poorer quality/AI generated photos of the person in different poses and backgrounds, then continue training with a lower learning rate and just the best photos of their face.

I was wondering if it would be possible to mark inputs, whether by file structure or file name and have the dataloader drop them from the training at the end of a certain epoch?

Proposed file structure:

train_woman
    |-1_sks woman
        |- Epoch 1
        |    |- 123.png
        |- Epoch 2
        |    |- 456.png
        |- 789.png (stays until the end)
